docs/examples/configuring-parallel-tests-with-code-climate.md

```diff
@@ -47,8 +48,8 @@ blocks:
           commands:
             - checkout
             # fetching partial tests from S3
-            - aws s3 sync coverage/ "s3://my_coverage_bucket/$SEMAPHORE_PROJECT_NAME/$SEMAPHORE_GIT_BRANCH/coverage/$SEMAPHORE_WORKFLOW_ID/"
+            - aws s3 sync "s3://my_coverage_bucket/$SEMAPHORE_PROJECT_NAME/$SEMAPHORE_GIT_BRANCH/coverage/$SEMAPHORE_WORKFLOW_ID/" coverage/
             # merging tests
-            - ./cc-test-reporter sum-coverage --output - --parts $(ls -1 coverage/ | wc -l) coverage/codeclimate.*.json 
+            - ./cc-test-reporter sum-coverage --output - --parts $(ls -1 coverage/ | wc -l) coverage/codeclimate.*.json > coverage/codeclimate.json
```

Comment:
The output of this command `./cc-test-reporter sum-coverage` is a string. So we fixed it by saving the output into `coverage/codeclimate.json`.